client/app/lib/components/sidebarstacksection/index.coffee

```diff
@@ -117,17 +116,18 @@ module.exports = class SidebarStackSection extends React.Component
         computeController.fetchStackTemplate templateId, (err, template) =>
           return @onMenuItemClickError 'making team default'  if err
 
-          computeController.makeTeamDefault template  unless err
+          computeController.makeTeamDefault template  if template
       when 'Share With Team'
         computeController.fetchStackTemplate templateId, (err, template) =>
           return @onMenuItemClickError 'sharing'  if err
-
-          computeController.setStackTemplateAccessLevel template, 'group'
+          if template
+            computeController.setStackTemplateAccessLevel template, 'group'
       when 'Make Private'
         computeController.fetchStackTemplate templateId, (err, template) =>
           return @onMenuItemClickError 'cloning'  if err
 
-          computeController.setStackTemplateAccessLevel template, 'private'
+          if template
+            computeController.setStackTemplateAccessLevel template, 'private'
```

Comment:
I think not having a `template` can be considered as `error` to give a proper feedback to user. WDYT?